hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestDynamoDBMetadataStore.java

```diff
@@ -281,22 +289,6 @@ private void deleteAllMetadata() throws IOException {
   public static void deleteMetadataUnderPath(final DynamoDBMetadataStore ms,
       final Path path, final boolean suppressErrors) throws IOException {
     ThrottleTracker throttleTracker = new ThrottleTracker(ms);
-    try (DurationInfo ignored = new DurationInfo(LOG, true, "prune")) {
```

Comment:
Why did you remove this? We do this operation on a separate testing table, just for this test, so it won't affect any other.